### PR TITLE
add Score variable in Game class, display score and alert user win wh…

### PR DESCRIPTION
**Pull request by wjduand**
…en the user gets 48 points
I just added #score id and getElementById to the html file, I copied and pasted the other two requests so

### CODE REVIEW

File: Assignment2/target/classes/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -153,10 +218,11 @@
 contentType:"application/json; charset=utf-8",
 dataType:"json",
 });
-
+document.getElementById("score").innerHTML = "Score: " + game.Score;
```
Comment (hammockt) [inline Assignment2/target/classes/views/AcesUp/AcesUp.flt.html:221]: Shouldn't this line be in the success part of the ajax call. Do you want to update the game's score if the post fails?

**Comment by wjduand** [comment]
But if the post fails the score won't change anyway.

File: Assignment2/target/classes/views/AcesUp/AcesUp.flt.html (hunk 1 above)
Comment (wjduand) [inline Assignment2/target/classes/views/AcesUp/AcesUp.flt.html:221]: But if the post fails the score won't change anyway.

Comment (wjduand) [inline Assignment2/target/classes/views/AcesUp/AcesUp.flt.html:221]: And in this way the "Score: " will always display

**Comment by hammockt** [comment]
But ajax calls are executed asynchronously. It's possible for the statement to execute before the ajax call returns successfully.

**Comment by wjduand** [comment]
But it will always display when the remove function is called if it is inside the remove function?

**Comment by hammockt** [comment]
You can always set the score paragraph to:

<p id="score">Score: 0</p>

This will guarantee that it is on the page (and appears to be 0 when it first loads).

**Comment by wjduand** [comment]
But it's not that necessary right? I mean this line only displays the score, even though user gets one point after remove a card but the score should just be an attribute.

**Comment by hammockt** [comment]
The statement can finish executing before the game is done removing a card. It's possible that the user can remove a card but the score won't update because the game variable didn't update in time. The success function in the ajax call updates the the game (and now the score). Having outside is risky.

**Comment by hammockt** [comment]
LGTM. I would merge but it has conflicts

**Comment by duffinp** [comment]
You will probably want to clone the current repo and recopy and paste your changes, and then just create a new pull request from that. That sounds easier than trying to open up an old one.

**Pull request closed without merging**